### ISSUE TITLE
Hard coded styling in 6.x

### ISSUE THREAD

**Issue by vincentparrett** [ActiveForums/ActiveForums#416]
I'm trying to upgrade my forums from 5.x to 6.x and I'm finding all sorts of issues with the theming. A bunch of hard coded elements using fontawesome are added in the code which can't be overridden. Why? This makes themes next to impossible. 

This is just one example 

`Template = Template.Replace("[DISPLAYNAME]", "<i class=\"fa fa-user fa-fw fa-blue\"></i>&nbsp;" + UserProfiles.GetDisplayName`

This hard codes an icon in the last post info. The code is now littered with this, everwhere, it has literally ruined the theme support. Calls to GetIcon have been replaced with hard coded markup with fontawesome.

**Comment by artfuld** [comment]
I agree. There's tons of html tags in the template that just shouldn't be there.
